test-data/stdlib-samples/3.2/test/test_genericpath.py

```diff
@@ -114,26 +114,21 @@ def test_getsize(self) -> None:
             support.unlink(support.TESTFN)
 
     def test_time(self) -> None:
-        f = open(support.TESTFN, "wb")
-        try:
-            f.write(b"foo")
-            f.close()
-            f = open(support.TESTFN, "ab")
-            f.write(b"bar")
-            f.close()
-            f = open(support.TESTFN, "rb")
-            d = f.read()
-            f.close()
-            self.assertEqual(d, b"foobar")
-
-            self.assertLessEqual(
-                self.pathmodule.getctime(support.TESTFN),
-                self.pathmodule.getmtime(support.TESTFN)
-            )
-        finally:
-            if not f.closed:
-                f.close()
-            support.unlink(support.TESTFN)
```

Comment:
Is it safe to drop the finally clause with unlink?

Reply:
It wouldn't, if this code were ever executed! We've talked about entirely getting rid of the stdlib samples before (e.g., in https://github.com/python/mypy/pull/8838), so I didn't try very hard to preserve them.